styles/dialog/fight.scss

```
@@ -19,13 +19,71 @@
         }
     }
 
+    .details {
+        display: grid;
+        grid-template-columns: repeat(4, 1fr);
+        border: 2px groove #aaaaaa;
+        justify-items: stretch;
+        column-gap: 2px;
+        row-gap: 2px;
+        .details-key{
+            display: flex;
+            align-items: center;
+            background: rgba(#000, .03);
+            border: 2px groove #aaaaaa;
+            padding: 1px;
+        }
+        .details-value{
+            display: flex;
+            align-items: center;
+            flex-direction: row-reverse;
+            justify-content: end;
+            text-align: end;
+            background: rgba(#444, .25);
+            border: 2px groove #aaaaaa;
+            text-align: end;
+            padding: 1px;
+        }
+    }
+    .weapon-block {
+        display: grid;
+        align-items: center;
+        padding: 5px 5px 0 5px;
+        //Weapon word has static width
+        grid-template-columns: [line1] 50px [line2] auto [col2-start];
+        justify-content: stretch;
+        .weapon-picker {
+            justify-self: end;
+            grid-column-start: 2;
+        }
```

Perhaps something simpler
```suggestion
        display: grid;
        width: 100%;
        padding: 5px 5px 0 5px;
        //Weapon word has static width
        grid-template-columns: 50px 1fr;
        .weapon-picker {
            text-align: end;
        }
```

---

template-columns and width are good,
but text-align doesn't handle everything well